test/general-specs.js

```diff
@@ -195,55 +195,4 @@ describe('general', function () {
         equal('mjpeg-consumer module is required to use MJPEG-over-HTTP features. Please install it with `npm i -g mjpeg-consumer`.');
     });
   }));
-
-  describe('OptionalPythonVersionCheck', withMocks({tp, utils}, (mocks) => {
```

Comment:
actually idb requires minimum Python 3.6 to be installed

Reply:
yeah, good point.
In this case, it was enough if `idb` command existed as this doctor, I thought.

If a user has no `idb` (or python 3.6), we'll guide them to https://github.com/appium/appium-idb . Then the user can see the python requirement. If the user already has idb, we can expect the user has python 3 env.